diff -Naurp a/amqp-1.7.0/tests/amqpexchange_unbind.phpt b/amqp-1.7.0/tests/amqpexchange_unbind.phpt
--- a/amqp-1.7.0/tests/amqpexchange_unbind.phpt	2016-04-26 09:00:45.000000000 -0500
+++ b/amqp-1.7.0/tests/amqpexchange_unbind.phpt	2016-06-21 14:58:17.989272753 -0500
@@ -1,7 +1,7 @@
 --TEST--
 AMQPExchange::unbind
 --SKIPIF--
-<?php if (!extension_loaded("amqp")) print "skip"; ?>
+<?php print "skip"; ?>
 --FILE--
 <?php
 $cnn = new AMQPConnection();
diff -Naurp a/amqp-1.7.0/tests/amqpexchange_unbind_with_arguments.phpt b/amqp-1.7.0/tests/amqpexchange_unbind_with_arguments.phpt
--- a/amqp-1.7.0/tests/amqpexchange_unbind_with_arguments.phpt	2016-04-26 09:00:45.000000000 -0500
+++ b/amqp-1.7.0/tests/amqpexchange_unbind_with_arguments.phpt	2016-06-21 14:59:02.484564817 -0500
@@ -1,7 +1,7 @@
 --TEST--
 AMQPExchange::unbind with arguments
 --SKIPIF--
-<?php if (!extension_loaded("amqp")) print "skip"; ?>
+<?php print "skip"; ?>
 --FILE--
 <?php
 $cnn = new AMQPConnection();
diff -Naurp a/amqp-1.7.0/tests/amqpexchange_unbind_without_key.phpt b/amqp-1.7.0/tests/amqpexchange_unbind_without_key.phpt
--- a/amqp-1.7.0/tests/amqpexchange_unbind_without_key.phpt	2016-04-26 09:00:45.000000000 -0500
+++ b/amqp-1.7.0/tests/amqpexchange_unbind_without_key.phpt	2016-06-21 14:59:16.131961113 -0500
@@ -1,7 +1,7 @@
 --TEST--
 AMQPExchange::unbind without key
 --SKIPIF--
-<?php if (!extension_loaded("amqp")) print "skip"; ?>
+<?php print "skip"; ?>
 --FILE--
 <?php
 $cnn = new AMQPConnection();
diff -Naurp a/amqp-1.7.0/tests/amqpexchange_unbind_without_key_with_arguments.phpt b/amqp-1.7.0/tests/amqpexchange_unbind_without_key_with_arguments.phpt
--- a/amqp-1.7.0/tests/amqpexchange_unbind_without_key_with_arguments.phpt	2016-04-26 09:00:45.000000000 -0500
+++ b/amqp-1.7.0/tests/amqpexchange_unbind_without_key_with_arguments.phpt	2016-06-21 14:59:24.596206897 -0500
@@ -1,7 +1,7 @@
 --TEST--
 AMQPExchange::unbind without key with arguments
 --SKIPIF--
-<?php if (!extension_loaded("amqp")) print "skip"; ?>
+<?php print "skip"; ?>
 --FILE--
 <?php
 $cnn = new AMQPConnection();
