diff -Naurp a/amqp-1.7.0/tests/bug_gh155_direct_reply_to.phpt b/amqp-1.7.0/tests/bug_gh155_direct_reply_to.phpt
--- a/amqp-1.7.0/tests/bug_gh155_direct_reply_to.phpt	2016-06-21 14:55:17.999046158 -0500
+++ b/amqp-1.7.0/tests/bug_gh155_direct_reply_to.phpt	2016-06-21 15:00:59.387959482 -0500
@@ -1,9 +1,7 @@
 --TEST--
 #155 RabbitMQ's Direct reply-to (related to consume multiple)
 --SKIPIF--
-<?php if (!extension_loaded("amqp")) {
-    print "skip";
-} ?>
+<?php print "skip"; ?>
 --FILE--
 <?php
 $conn = new AMQPConnection();
