diff -Naurp a/amqp-1.7.0/tests/amqpchannel_basicRecover.phpt b/amqp-1.7.0/tests/amqpchannel_basicRecover.phpt
--- a/amqp-1.7.0/tests/amqpchannel_basicRecover.phpt	2016-04-26 09:00:45.000000000 -0500
+++ b/amqp-1.7.0/tests/amqpchannel_basicRecover.phpt	2016-06-21 14:55:53.821086371 -0500
@@ -1,11 +1,7 @@
 --TEST--
 AMQPChannel::basicRecover
 --SKIPIF--
-<?php
-if (!extension_loaded("amqp") || version_compare(PHP_VERSION, '5.3', '<')) {
-    print "skip";
-}
-?>
+<?php print "skip"; ?>
 --FILE--
 <?php
 $time = microtime(true);
